labs/vstsextend/sonarcloud/readme.md

```diff
@@ -14,9 +15,9 @@ Technical debt is the set of problems in a development effort that make forward
 [SonarCloud](https://about.sonarcloud.io/){:target="_blank"} is the code quality cloud service provided by SonarSource.
```

Comment:
All of the other occurrences of `target="_blank"` have been changed to `target="\_blank"` except this one. Is that deliberate?

Reply:
Yes, that was to test because the rendering failed to do something with that (it's displayed in raw on the github rendering).

Reply:
This has been corrected